src/autodiff_model.jl

```diff
@@ -114,6 +114,13 @@ function jac(nlp :: ADNLPModel, x :: AbstractVector)
   return ForwardDiff.jacobian(nlp.c, x)
 end
 
+function jac_coord(nlp :: ADNLPModel, x :: AbstractVector)
+  Jx = jac(nlp, x)
+  m, n = nlp.meta.ncon, nlp.meta.nvar
+  I = ((i,j) for i = 1:m, j = 1:n)
+  return (getindex.(I, 1)[:], getindex.(I, 2)[:], Jx[:])
```

Comment:
Julia 0.7 is failing because `I` is a generator instead of an array. It's a small cost, but it could be reason to drop 0.7. Since we already made the transition from 0.6, it shouldn't affect anything.

Reply:
I agree but I still find 0.7 useful in the sense that it gives warnings and tells you how to change your code to make it compatible with 1.0. Could we just add an `if VERSION ...` for the moment?